config/kovan-addresses.json

```diff
@@ -42,17 +42,16 @@
   "PIP_USDC": "0x4c51c2584309b7BF328F89609FDd03B3b95fC677",
   "MCD_JOIN_USDC_A": "0x4c514656E7dB7B859E994322D2b511d99105C1Eb",
   "MCD_FLIP_USDC_A": "0x45d5b4A304f554262539cfd167dd05e331Da686E",
+  "MCD_JOIN_USDC_B": "0xaca10483e7248453BB6C5afc3e403e8b7EeDF314",
+  "MCD_FLIP_USDC_B": "0x93AE217b0C6bF52E9FFea6Ab191cCD438d9EC0de",
   "WBTC": "0x7419f744bBF35956020C1687fF68911cD777f865",
   "PIP_WBTC": "0x2f38a1bD385A9B395D01f2Cbf767b4527663edDB",
   "MCD_JOIN_WBTC_A": "0xB879c7d51439F8e7AC6b2f82583746A0d336e63F",
   "MCD_FLIP_WBTC_A": "0xc45A1b76D3316D56a0225fB02Ab6b7637403fF67",
-  "SAI": "0xC4375B7De8af5a38a93548eb8453a498222C4fF2",
```

Comment:
I suppose we're removing SAI addresses from the Kovan deployment because there's no reason to test anything with SAI anymore?

Reply:
Yeah, I am making the same assumption here.  The addresses are removed from `1.0.7` though.

Reply:
I should say, in `kovan`.  The house cleaning executive has not been deployed to `mainnet` yet, so those addresses are likely going to remain in the changelog for `mainnet`.